### PR TITLE
✨ Load instance settings with edge function

### PR DESCRIPTION
**Pull request by Koncopd**
https://github.com/laminlabs/laminhub/pull/1209

### CODE REVIEW

**Comment by Koncopd** [comment]
@falexwolf i think this is ready, along this https://github.com/laminlabs/laminhub/pull/1209

File: lamindb_setup/core/_settings_store.py [hunk 1]
```diff
@@ -49,6 +49,7 @@ def system_storage_settings_file():
 
 
 class InstanceSettingsStore(BaseSettings):
+    api_url: Optional[str] = None
```
Comment (falexwolf) [inline lamindb_setup/core/_settings_store.py:52]: We also need the `schema_id`, here, don't we?

For `laminr` ([WP](https://www.notion.so/laminlabs/Build-an-R-SDK-laminR-8efbfd00157b4efe91b0927730492c0d)).

**Comment by falexwolf** [comment]
This looks great to me!

It'd be nice though if this was starting to go live for the lamin team so that we build confidence in it. We could check for whether a user handle is in `sunnyosun`, `Koncopd`, `zethson`, `falexwolf` and then call the new connect function instead of the old one. Can you make this change.

Note: For laminr, we'll need to document how to call this edge function from within R. But that's probably for Notion.

File: lamindb_setup/core/_hub_core.py [hunk 2]
```diff
@@ -367,6 +367,61 @@ def _connect_instance(
     return instance, storage  # type: ignore
 
 
+def _connect_instance_remote(
```
Comment (falexwolf) [inline lamindb_setup/core/_hub_core.py:370]: Can you call it `_connect_instance_new()` instead?

And then we'll delete the existing `_connect_instance` function once we spend 1-2 weeks testing it.

**Review by falexwolf — CHANGES_REQUESTED** [review]
Thanks, Sergei! 3 small changes, please.

File: lamindb_setup/core/_settings_store.py (hunk 1 above)
Comment (Koncopd) [reply lamindb_setup/core/_settings_store.py:52]: Added.

**Comment by Koncopd** [comment]
>It'd be nice though if this was starting to go live for the lamin team so that we build confidence in it. We could check for whether a user handle is in sunnyosun, Koncopd, zethson, falexwolf and then call the new connect function instead of the old one. Can you make this change.

This i will add to `ln.connect` in a new PR when the edge function is deployed.

**Comment by falexwolf** [comment]
Ok! Agreed!

File: lamindb_setup/core/_hub_core.py (hunk 2 above)
Comment (Koncopd) [reply lamindb_setup/core/_hub_core.py]: done

**Pull request merged**